.circleci/config.yml

```diff
@@ -55,20 +55,21 @@ jobs:
           paths:
             - occollector_linux
 
-  publish:
+  merge-publish:
     executor: gradle_docker
     steps:
-      - add_ssh_keys: &ssh-fingerprint
-          fingerprints:
-            - "4b:17:55:41:7c:82:34:17:ff:2a:6b:02:cf:6f:86:25"
       - setup_build_environment
-      - setup_remote_docker
+      - setup_remote_docker: &latest_remote_docker
+          version: 19.03.12
+      - docker_login
+      - run: ./gradlew dockerPushImages
+  release-publish:
+    executor: gradle_docker
+    steps:
+      - setup_build_environment
+      - setup_remote_docker: *latest_remote_docker
       - docker_login
-      - run: ./gradlew :tag -Prelease
-      - run: cp -v /tmp/workspace/occollector_linux ./cmd/occollector/
```

Comment:
I'm not spying this copy command in the new version, is it still needed?

Reply:
ohh. I guess we do need it. it has dockerfile so yeah.